README.adoc

```diff
@@ -2537,18 +2544,29 @@ class Person
 end
 ----
 
-==== Non-declarative Methods That Have "keyword" Status in Ruby [[method-invocation-parens-non-declarative-keyword]]
+===== Non-Declarative Methods That Have "keyword" Status in Ruby [[method-invocation-parens-non-declarative-keyword]]
+
+There are two schools of thought within the community: one says to always omit parentheses for non-declarative methods that have "keyword" status (e.g., `Kernel` instance methods), and the other says not to omit them.
 
-Can omit parentheses for methods that have "keyword" status in Ruby, but are not declarative:
+The omit-parentheses school says that certain non-declarative methods feel like they have "keyword" status.
+Since they feel like keywords, the code reads better *without* parentheses because it looks like keywords rather than method calls.
+
+The use-parentheses school says that there's nothing special about these non-declarative methods, their semantics are ordinary, and which methods have "keyword" status is not well-defined.
```

Comment:
Probably I'd stick to the more common style (omitting parens) and mention the alternative in a nice callout block.

Reply:
OK, I force-pushed an amended commit that doesn't go so far as to stick with the omitting-parens style and only mentioning the alternative in a callout block, but it does make it clear that one is much more popular than the other.